### PR TITLE
Spark post uninstall notes

### PR DESCRIPTION
**Pull request by tnachen**

### CODE REVIEW

**Comment by elingg** [comment]
LGTM!

File: repo/packages/S/spark/0/package.json [hunk 1]
```diff
@@ -12,5 +12,12 @@
...
-  "postInstallNotes": "Spark DCOS Service has been successfully installed!\nNote that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service.\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK"
+  "postInstallNotes": "Spark DCOS Service has been successfully installed!\nNote that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service.\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK",
+  "postUninstallNotes": "The Apache Spark DCOS Service has been uninstalled and will no longer run.\nPlease follow the instructions at http://beta-docs.mesosphere.com/services/spark/#uninstall to clean up any persisted state.",
```
Comment (elingg) [inline repo/packages/S/spark/0/package.json]: Note that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service. - this should be in the preInstallNotes

Comment (elingg) [inline repo/packages/S/spark/0/package.json]: postInstallNotes should have Spark DCOS Service has been successfully installed!\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK

**Comment by elingg** [comment]
LGTM! I left a couple of comments to make it consistent with other services. Also, resource requirements should only be in the preinstallNotes along with letting people know that the service is beta.

**Comment by tnachen** [comment]
Thx @elingg! Addressed your comments.

**Pull request merged**